spec/fabricators/match_fabricator.rb

```diff
@@ -0,0 +1,6 @@
+Fabricator(:match) do
+  home_team Team.all.ids.sample
```

Comment:
better here to just fabricate a team. you may not have any existing to sample from when the test runs.